pixi.toml

```
@@ -4,9 +4,12 @@ version = "0.1.0"
 description = "Reader to load data from DICOM files into Apache Arrow"
 authors = ["Marc Garcia <garcia.marc@gmail.com>"]
 channels = ["conda-forge"]
-platforms = ["linux-64"]
+platforms = ["linux-64", "osx-arm64"]  # --> add in "osx-arm64" for MacOS
 
 [tasks]
-main = "LD_LIBRARY_PATH=target/debug python dicom_reader/__init__.py"
+main = "DYLD_LIBRARY_PATH=target/debug python dicom_reader/__init__.py"   
```

I think we can just have both variables so it works in both OSs.

```suggestion
main = "LD_LIBRARY_PATH=target/debug DYLD_LIBRARY_PATH=target/debug python dicom_reader/__init__.py"   
```

---

Done